css/style.css

```diff
@@ -59,18 +67,140 @@ body {
 
 /* Center the container */
 #container {
-  width: 960px;
   margin: auto;
+  padding-right: 20px;
+  padding-left: 40px;
+
 }
 
-header {
+div{
+
+  background-color: #033048;
```

Comment:
Also, rather than styling every `div` to have this background color, I recommend just setting the background color on `.topbar`